4 - AnimalShelter/Tests/AnimalShelter_GetRemoveTests.cs

```diff
@@ -0,0 +1,292 @@
+﻿using System.Collections.Generic;
+using AnimalShelterCode;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+
+namespace AnimalShelterTests
+{
+    [TestClass]
```

Comment:
You did a really nice job with your unit testing, this will go a long way to empower your development career and ability.